### DISCUSSION TITLE
Error al crear un archivo .h

### DISCUSSION THREAD

**Discussion by tadonisb** [IIC2133-PUC/2024-1#14] [Ayuda] [answered]
Hola! No sé por qué al momento de crear headers, VSC me arroja el error

"command-line error: language modes specified are incompatible"

dejo una imagen también de referencia. Inicialmente pensé que el problema eran las importaciones, después las estructuras que estaba creando, pero al final terminé por borrar todo el código del header y el error seguía marcando. No puedo encontrar nada similar en internet.

![Screenshot (846)](https://github.com/IIC2133-PUC/2024-1/assets/69985137/25c9c98e-e51d-4fd0-a6a8-664cbbb11f68)

**Comment by CataAlamos** [comment, marked as answer]
Hola! Descargaste alguna extensión de C en VSC? Intentaste 
https://www.reddit.com/r/vscode/comments/16cde8i/commandline_error_language_modes_specified_are/
